HW0_enriquezvazquezs.py

```diff
@@ -0,0 +1,121 @@
+#################################################################################
+# Author: Sandy Enriquez Vazquez
+# Username: enriquezvazquezs
+#
+# Assignment:HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: Draw something complex
+# Google Doc Link: https://docs.google.com/document/d/16tdyTo7Va4O3NHu80p5LXP_pasdFz-eTVAOKTULSqdg/edit?tab=t.0#heading=
+# h.j4414z2ufr72
+#
+#################################################################################
+# Acknowledgements:OpenAI's ChatGPT. (2025). Example of Python code to draw a flower with five petals using turtle.
+# Retrieved from: https://chat.openai.com
+# Example given:
+#import turtle
+
+# Set up the turtle screen
+#screen = turtle.Screen()
+#screen.bgcolor("white")
+
+# Create the turtle object
+#flower = turtle.Turtle()
+#flower.shape("turtle")
+#flower.speed(10)  # Fast drawing speed
+
+# Function to draw a single petal
+#def draw_petal():
+#   flower.color("red")
+#   flower.circle(100, 60)  # Draw a circle with 100 radius and 60 degrees arc
+#   flower.left(120)  # Turn the turtle to the left by 120 degrees
+#   flower.circle(100, 60)
+#   flower.left(120)
+
+# Draw five petals
+#for _ in range(5):
+#    draw_petal()
+#    flower.left(72)  # Turn the turtle to make space for the next petal
+
+# Hide the turtle after drawing
+#flower.hideturtle()
+
+# Keep the window open until it is closed by the user
+#turtle.done()
+
+# *I couldn't figure out how to get the exact link to the page*
+#
+#################################################################################
+
+import turtle
+
+def draw_grass(g):
+    """
+    draws grass
+    :param g:
```

Comment:
Here is where you indicate what `g` is. This should probably say "A turtle object" or similar.